tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreManagedLedgerOffloader.java

```diff
@@ -94,7 +94,7 @@
     final private long maxBufferLength;
     final private ConcurrentLinkedQueue<Entry> offloadBuffer = new ConcurrentLinkedQueue<>();
     private CompletableFuture<OffloadResult> offloadResult;
-    private volatile PositionImpl lastOfferedPosition = PositionImpl.latest;
+    private volatile PositionImpl lastOfferedPosition = PositionImpl.earliest;
```

Comment:
Why do you need to change this?

Reply:
This field is introduced at https://github.com/apache/pulsar/pull/9096, but actually first used here, in actual usage, the original value will not be used, I just initialize it here with a make sense value to avoid the compiler warning.